src/layouting/layouting.jl

```diff
@@ -176,15 +176,15 @@ ispixelcam(x) = false
 """
     vbox(scenes...; parent = Scene(clear = false), kwargs...)
 
-Lay the given Scenes out on the vertical axis.  For example, two Scenes `vbox`ed
+Attach the given Scenes together on the vertical axis.  For example, two Scenes `vbox`ed
```

Comment:
```suggestion
Box the scenes together on the vertical axis.  For example, two Scenes `vbox`ed
```

The reason I say this is that I don't want to give the wrong impression - there's no attachment per se, at least not yet.